tools/client_package_initializer.py

```diff
@@ -148,11 +171,12 @@ def build(self) -> None:
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description="Swagger package builder")
+    parser.add_argument("client_extra_files_dir", help="External modules to add to the python api client", type=str)
```

Comment:
any ``*extra*`` argument should come after the main one :) 
also, I recommend changing to named argument as its role is not clear when looking usage in ``hack/generate.sh``

Reply:
It's not extra argument. I can change the order but it's mandatory argument.

Reply:
yeah, reorder
base always comes before extra :)